libs/paymentProcessor.js

```diff
@@ -69,23 +69,6 @@ function SetupForPool(logger, poolOptions, setupFinished){
                     logger.error(logSystem, logComponent, 'Error with payment processing daemon ' + JSON.stringify(result.error));
                     callback(true);
                 }
-                else if (!result.response || !result.response.ismine) {
-                            daemon.cmd('getaddressinfo', [poolOptions.address], function(result) {
-                        if (result.error){
-                            logger.error(logSystem, logComponent, 'Error with payment processing daemon, getaddressinfo failed ... ' + JSON.stringify(result.error));
-                            callback(true);
-                        }
-                        else if (!result.response || !result.response.ismine) {
-                            logger.error(logSystem, logComponent,
-                                    'Daemon does not own pool address - payment processing can not be done with this daemon, '
-                                    + JSON.stringify(result.response));
-                            callback(true);
-                        }
-                        else{
-                            callback()
-                        }
-                    }, true);
-                }
```

Comment:
Why were these checks removed?

Reply:
there are no "getaddressinfo" in BitcoinSubsidium daemon